client/src/pages/Login.js

```diff
@@ -283,6 +338,7 @@ const Login = ({ isLoginForm }) => {
                   style={StyleInput}
                 />
               </InputWrapper>
+              <ErrorMessage id="email-error"></ErrorMessage>
```

Comment:
As mentioned previously, one way to approach this is to toggle `ErrorMessage` if there are errors.